error/s2n_errno.c

```diff
@@ -14,155 +14,191 @@
  */
 
 #include <strings.h>
+#include <stdio.h>
 #include <stdlib.h>
-
+#include <string.h>
 #include "error/s2n_errno.h"
 
 #include <s2n.h>
+#include "utils/s2n_map.h"
+#include "utils/s2n_safety.h"
 
 __thread int s2n_errno;
 __thread const char *s2n_debug_str;
 
 static const char *no_such_language = "Language is not supported for error translation";
 static const char *no_such_error = "Internal s2n error";
 
-struct s2n_error_translation {
+typedef struct _s2n_error_translation {
```

Comment:
I don't think any other structs in s2n use typedef, any reasons for changing it here?

Reply:
I just do not want to write struct everytime :) I can change to s2n style though.